### PR TITLE
Lps 139835

### PR DESCRIPTION
**Pull request by DL-EthanSun**

### CODE REVIEW

**Comment by DL-EthanSun** [comment]
ci:test:gradle-plugins

**Comment by DL-EthanSun** [comment]
ci:test:gradle-plugins

File: modules/sdk/gradle-plugins-workspace/build.gradle [hunk 1]
```diff
@@ -91,7 +91,7 @@ dependencies {
 	compile group: "com.bmuschko", name: "gradle-docker-plugin", version: "6.6.1"
 	compile group: 'com.google.code.gson', name: 'gson', version: '2.8.6'
 	compile group: "com.liferay", name: "com.liferay.ant.bnd", version: "3.2.7"
-	compile(group: "com.liferay", name: "com.liferay.gradle.plugins", version: "13.2.57") {
+	compile(group: "com.liferay", name: "com.liferay.gradle.plugins", version: "13.2.58") {
```
Comment (lawrence-lee) [inline modules/sdk/gradle-plugins-workspace/build.gradle:94]: This needs a new commit and new publish of workspace plugin

File: modules/sdk/gradle-plugins-defaults/build.gradle [hunk 2]
```diff
@@ -18,7 +18,7 @@ copyGradleTestDependencies {
 dependencies {
 	compile group: "com.github.jk1", name: "gradle-license-report", version: "1.8"
 	compile group: "com.gradle.publish", name: "plugin-publish-plugin", version: "0.11.0"
-	compile group: "com.liferay", name: "com.liferay.gradle.plugins", version: "13.2.57"
+	compile group: "com.liferay", name: "com.liferay.gradle.plugins", version: "13.2.58"
```
Comment (lawrence-lee) [inline modules/sdk/gradle-plugins-defaults/build.gradle:21]: This needs a new publish of gradle-plugins-defaults. please put this in separate commit

File: modules/sdk/gradle-plugins-wsdd-builder/src/gradleTest/smoke/build.gradle [hunk 3]
```diff
@@ -28,7 +28,7 @@ project(":sample-api") {
 	dependencies {
 		compile group: "biz.aQute.bnd", name: "biz.aQute.bndlib", version: "3.1.0"
 		compile group: "com.liferay", name: "com.liferay.osgi.util", version: "3.0.0"
-		compile group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "2.6.0"
+		compile group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "4.81.2"
```
Comment (lawrence-lee) [inline modules/sdk/gradle-plugins-wsdd-builder/src/gradleTest/smoke/build.gradle:31]: Do we need this change in order for the tests to pass? seems like quite a big jump from 2.6.0 to 4.81.2

Comment (DL-EthanSun) [reply modules/sdk/gradle-plugins-wsdd-builder/src/gradleTest/smoke/build.gradle:31]: It's necessary to upgrade com.liferay.portal.kernel to at least 4.40.0(I thought the 4.81.2 looks stable, so I chose it), because the gradle test file in  gradle-plugins-wsdd-builder need com.liferay.petra.string.StringBundler rather than com.liferay.portal.kernel.util.StringBundler and the import dependency of BasePersistenceImpl.java  has changed com.liferay.portal.kernel.util.StringBundler to com.liferay.petra.string.StringBundler until this commit 121b75ef0676be297fa5b421995c5999222e10c5 pushed, so we need upgrade com.liferay.portal.kernel that contain this commit to pass the gradleTest, otherwise it would occurred the cast problem.

Comment (DL-EthanSun) [reply modules/sdk/gradle-plugins-wsdd-builder/src/gradleTest/smoke/build.gradle:31]: Okay, no problem
